src/server/land-grants/parcel/land-parcel.html

```diff
@@ -11,17 +11,12 @@
 <div class="govuk-body">
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      {% if errors.length > 0 %}
-        {% set errorList = [] %}
-        {% for error in errors %}
-            {% set errorList = errorList.concat([{
-              text: error,
-              href: ''
-            }]) %}
-        {% endfor %}
+      {% if selectParcelError %}
         {{ govukErrorSummary({
           titleText: "There is a problem",
-          errorList: errorList
+          errorList: [{
+            text: 'Please select a land parcel from the list'
```

Comment:
Could we not add this error message in the controller? It would make it alot easier to unit test the controller to ensure that the error message is sent to the view if the data is invalid